src/Stripe.net/Services/TaxIds/TaxIdService.cs

```diff
@@ -67,5 +67,12 @@ public virtual IEnumerable<TaxId> ListAutoPaging(string customerId, TaxIdListOpt
         {
             return this.ListNestedEntitiesAutoPaging(customerId, options, requestOptions);
         }
+
+#if !NET45
+        public virtual IAsyncEnumerable<TaxId> ListAutoPagingAsync(string customerId, TaxIdListOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default)
```

Comment:
Wondering if these `customerId` parameter names will be easy to derive from the spec, or if it would be easier to use `id` or `parentId` in the nested cases...

Reply:
They shouldn't need to. In another PR for your codegen branch we renamed all of those to `parentId` and `id` instead. It's likely just that ob used a codemod or a regex to generate those

Reply:
Yep, this PR is like 90% codemod. Sorry for the additional toil CJ :/